doc/source/basics.rst

```diff
@@ -1755,12 +1755,14 @@ then the more *general* one will be used as the result of the operation.
    # conversion of dtypes
    df3.astype('float32').dtypes
 
+.. versionadded:: 0.20.0
```

Comment:
@m-charlton As I said above, this feature was already introduced in 0.19.0. That's the reason I asked to change 0.20.0 to 0.19.0 in the versionadded here